### DISCUSSION TITLE
How to add a new Country Map using Superset Own docker Hub?

### DISCUSSION THREAD

**Discussion by JoaquinPretell92** [apache/superset#18324] [Q&A / Help]
Hello,

I run superset on version 1.0.1 and i want to add my country map to it. Since i call the image from your Docker Hub repository, i understand i have to download the image and make a custom one.
The question is where do i have to upload my geojson.
From this documentation i read i have to put it in here superset/assets/src/visualizations/CountryMap/countries, but thats a path for a virtual machine.
https://apache-superset.readthedocs.io/en/0.35.2/visualization.html#need-to-add-a-new-country

I want to know if someone have the steps to do it or maybe i can make a feature requests so they can add it with the geojson i make.

Regards.

**Comment by junlincc** [comment]
Hi! you can take a look at those PRs as reference :)  you do have to add your file in superset-ui repo though. hope it helps! 
https://github.com/apache-superset/superset-ui/pull/1334
https://github.com/apache-superset/superset-ui/pull/1319
https://github.com/apache-superset/superset-ui/pull/1267

**Comment by JoaquinPretell92** [comment]
Hello @junlincc,

Thanks for the information.
I have another question, from what I see, I would also have to update the version of my superset to the lastest since the PR go to master directly or its possible to ask to PR to an old version like mine (1.0.1)

Regards.
